app/views/devise/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= I18n.t("errors.messages.not_saved",
```

Comment:
I think  it works without `I18n` part
```suggestion
      <%= t("errors.messages.not_saved",
```

Reply:
Do you know why it works in views while it does not work if I just call "t()" in my RSpec tests? :)
Thanks,

Reply:
Because it is a part of helpers, which are only available in view https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/TranslationHelper.html. You can try the following command by opening up rails console. 

```
   rails c (open up rails console)

   include ActionView::Helpers::TranslationHelper
   t("errors.messages.not_saved")
```

Reply:
Thanks! Very clear